lib/datadog/tracing/configuration/settings.rb

```diff
@@ -380,8 +385,11 @@ def self.extended(base)
               # This option is recommended for internal use only.
               #
               # @default `{}`
-              # @return [Hash,nil]
-              option :writer_options, default: ->(_i) { {} }
+              # @return [Hash]
+              option :writer_options do |o|
+                o.type :hash
```

Comment:
This was changed to not nillable, but after inspecting our codebase, this change has no impact as the default `{}` is accepted everywhere `nil` would.